src/reducers/platform.js

```diff
@@ -24,21 +24,23 @@ export function app(state=DEFAULT_APP_STATE, action={}) {
 
 	switch (action.type) {
 	case 'API_REQUEST':
-		if (action.meta.logout) {
+		if ((action.meta || {}).logout) {
 			return DEFAULT_APP_STATE;  // clear app state during logout
 		}
-		return state;
-	case 'CACHE_SUCCESS':  // fall through - same effect as API success
+		return { ...state, isFetching: true };
 	case 'API_SUCCESS':
-		// API_SUCCESS contains an array of responses, but we just need to build a single
+		state.isFetching = false;  // fall through - everything else is the same as CACHE_SUCCCESS
```

Comment:
make sure you use the split view in github for this section of code - the 'Unified' version has some weird change groupings